### PR TITLE
Possibly fix Door Opened notifications?

### PR DESCRIPTION
**Pull request by thinkl33t**
I think this is why the Door Opened notifications aren't getting through due to now being all unicode and shit.

Basically *payloads* from mqtt are received as binary, and python 3 strings are proper unicode strings.  So its checking if a binary string is the same as a unicode string, and its failing because they aren't

### CODE REVIEW

**Review by BinaryKitten — APPROVED** [review]

**Review by fahadsadah — APPROVED** [review]
Great idea, thank you! Code looks great, you haven't missed any :D 

Topics get pre-decoded by the library so those don't need done \o/

File: bot.py [hunk 1]
```diff
@@ -81,11 +81,11 @@ def on_message(mosq, obj, msg):
         end_to_bot("Unknown card at door", increment = True)
```
Comment (conorriches) [inline bot.py]: could "end_to_bot" be causing it to fall over?

**Review by conorriches — COMMENTED** [review]
quick question about a function I'm not sure isn't a typo

File: bot.py (hunk 1 above)
Comment (fahadsadah) [reply bot.py]: Very likely! This is fixed in master and merged back into this PR :)

**Pull request merged**